0000-miner-update.md

```diff
@@ -0,0 +1,199 @@
+- Start Date: 2020-02-13
+- HIP PR: <!-- leave this empty -->
+- Tracking Issue: <!-- leave this empty -->
+
+# Summary
```

Comment:
I think this Summary exposes the problem and its scope well, but I think it would be helpful to have a summary of proposed solution.

Reply:
Done.